src/init/variables/computers/AssetPathResolver.ts

```diff
@@ -0,0 +1,25 @@
+import { resolveAssetPath } from '../../../util/PathUtil';
+import { ValueComputer } from './ValueComputer';
+
+/**
+ * A {@link ValueComputer} that converts a path value to an absolute asset path by making use of `resolveAssetPath`.
+ * Returns the default path in case it is defined and no path was found in the map.
+ */
+export class AssetPathResolver extends ValueComputer {
+  private readonly key: string;
+  private readonly defaultPath?: string;
+
+  public constructor(key: string, defaultPath?: string) {
+    super();
+    this.key = key;
+    this.defaultPath = defaultPath;
```

Comment:
`defaultValue` for consistency? And perhaps have the above class as base class?

Reply:
I think the difference in parameter name makes it clearer that the `defaultPath` value still is going to be interpreted.

Can subclass but I do wonder if the 2 lines of code it saves is worth the extra mental load of having another layer in the hierarchy when browsing the code (which is also quite minor I admit).